.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+Thanks for participating in Jupyter governance ✨
+Below are a few guidelines for making your proposed change as efficient and productive
+as possible.
+
+## Questions to answer ❓
```

Comment:
Wow, TIL about another whole area of unicode characters. "red question mark" really is a thing. As well as things like ⁉️ and ❣️